examples/website/map-tile/app.js

```diff
@@ -15,6 +15,21 @@ const INITIAL_VIEW_STATE = {
   bearing: 0
 };
 
+const COPYRIGHT_LICENCE_STYLE = {
```

Comment:
Should we standardize on American English? `license`

Reply:
Fixed.